link-parent/link-ilp-over-http/src/main/java/org/interledger/link/http/IlpOverHttpConstants.java

```diff
@@ -12,6 +12,6 @@
   String OCTET_STREAM_STRING = OCTET_STREAM.toString();
   MediaType APPLICATION_OCTET_STREAM = MediaType.parse(OCTET_STREAM_STRING);
 
-  String BEARER = "Bearer ";
+  String BEARER = "Bearer";
```

Comment:
You'll need to search through the code-base for usage of `BEARER` and add a space in there. This is why the build is breaking.